utils/dateUtils.ts

```diff
@@ -1,15 +1,24 @@
-export function isSameDay(date1: Date, date2: Date): boolean {
-    return (
-        date1.getDate() === date2.getDate() &&
-        date1.getMonth() === date2.getMonth() &&
-        date1.getFullYear() === date2.getFullYear()
-    );
-}
-
 export function formatSessionTime(dateTimeStr: string) {
-    const date = new Date(dateTimeStr);
-    const hours = String(date.getUTCHours()).padStart(2, "0");
-    const minutes = String(date.getUTCMinutes()).padStart(2, "0");
+    let date;
+    if (dateTimeStr.includes('AM') || dateTimeStr.includes('PM')) {
+        const [time, period] = dateTimeStr.split(' ');
+        const [hoursStr, minutesStr] = time.split(':');
+        let hours = parseInt(hoursStr);
+        const minutes = parseInt(minutesStr);
+
+        if (period === 'PM' && hours < 12) {
```

Comment:
Isn't this possible by library methods on datetime like python?

Reply:
I don't think so 😐